packages/mds-transactions-service/service/provider.ts

```diff
@@ -18,45 +23,78 @@ export const TransactionServiceProvider: ServiceProvider<TransactionService> & P
   },
   createTransactions: async transactions => {
     try {
-      return ServiceResult(await TransactionRepository.createTransactions(transactions.map(validateTransactionDomainModel)))
+      return ServiceResult(
+        await TransactionRepository.createTransactions(transactions.map(validateTransactionDomainModel))
+      )
     } catch (error) /* istanbul ignore next */ {
       const exception = ServiceException('Error Creating Transactions', error)
       logger.error(exception, error)
       return exception
     }
   },
-  getTransaction: async name => {
+  getTransaction: async (transaction_id: UUID) => {
```

Comment:
TS can infer the argument types in `ServiceProvider` methods.
```suggestion
  getTransaction: async transaction_id => {
```